src/instructlab/model/backends/backends.py

```diff
@@ -25,8 +25,6 @@
 from ...configuration import get_api_base
```

Comment:
Looks like you're missing an import for `VLLM` and `LLAMA_CPP` here from configuration